massa-versioning/src/versioning.rs

```diff
@@ -908,11 +908,32 @@ impl MipStoreRaw {
                 {
                     let entry_value = e.get_mut();
                     *entry_value = entry_value.saturating_sub(1);
+                    network_version_count = *entry_value;
                     if *entry_value == 0 {
                         self.stats.network_version_counters.remove(&removed_version);
                     }
                 }
             }
+
+            if announced_network_version != 0 {
+                let vote_ratio = Ratio::new(
+                    network_version_count,
+                    self.stats.config.block_count_considered as u64,
+                );
+
+                if vote_ratio > MIP_STATS_RATIO_WARN_ANNOUNCED_VERSION {
+                    let last_key_value = self.store.last_key_value();
+                    if let Some((mi, _ms)) = last_key_value {
+                        if announced_network_version > mi.version {
+                            // Vote ratio is > 30%
+                            // announced version is not known (not in MIP store)
+                            // announced version is > to the last known network version in MIP store
+                            // -> Warn the user to update
+                            warn!("Please update your node if you want to support this update (new network version: {})", announced_network_version);
```

Comment:
we can make the message a bit more informative: `XXX our of YYY last blocks advertised that they are willing to transition to version ZZZ. You should update your node if you wish to move to that version.`

Reply:
Done